docs/guide/results/working_with_networks.rst

```diff
@@ -3,8 +3,56 @@
 Working with networks of results
 ================================
 
-issue #803
+After running a **network** of free energy calculations,
+we often want to consider analysing the corresponding network of results.
 
+.. _userguide_MLE:
 
-The :ref:`openfe gather <cli_gather>` command offers a way to collate information across many different individual
+Converting relative results to absolute estimates
+-------------------------------------------------
+
+When a network of relative free energies have been calculated,
+a commonly performed task is to transform these pairwise estimations of **relative** free energies
+into a **absolute** free energies.
+This is done using a maximum likelihood estimator (MLE) [1]_,
+as implemented in the `cinnabar`_ package.
+This approach uses the matrix of relative pairwise measurements and their uncertainties,
+to estimate the overall ranking of
+To use this approach the network of pairwise measurements needs to be fully connected,
+i.e. there should be a way to trace a path along pairwise measurements between any two nodes (ligands) on the network.
+
+.. note::
+   The results of a MLE estimation will have a **mean** of 0.0,
```

Comment:
Do we have a code snippet somewhere (maybe in a cookbook?) that shows how one can shift the data?

Reply:
not really, I need to add in this shifting to the cinnabar docs (and our gather command probably...)